src/app_service/service/ens/service.nim

```diff
@@ -7,6 +7,7 @@ import ../../../app/core/eventemitter
 import ../../../app/core/tasks/[qt, threadpool]
 
 import ../../../backend/eth as status_eth
+import ../../../backend/ens as status_ens
 
 import ../../common/conversion as common_conversion
 import utils as ens_utils
```

Comment:
Is the service full migrated, I can still see function about fetching expiration time for example?

Reply:
The ens service should not make any call via eth_call but only to ens backend, I think @saledjenic linked another issue to the PR.

Reply:
Thanks for the head up, we discussed that in our cohort chat.